cyfra-vulkan/src/main/scala/io/computenode/cyfra/vulkan/executor/AbstractExecutor.scala

```diff
@@ -7,11 +7,10 @@ import io.computenode.cyfra.vulkan.core.Device
 import io.computenode.cyfra.vulkan.memory.{Allocator, Buffer, DescriptorPool, DescriptorSet}
 import org.lwjgl.BufferUtils
 import org.lwjgl.util.vma.Vma.VMA_MEMORY_USAGE_UNKNOWN
+import org.lwjgl.util.vma.Vma.VMA_MEMORY_USAGE_GPU_TO_CPU
 import org.lwjgl.vulkan.*
 import org.lwjgl.vulkan.VK10.*
 
-import java.nio.ByteBuffer
-
 private[cyfra] abstract class AbstractExecutor(dataLength: Int, val bufferActions: Seq[BufferAction], context: VulkanContext) {
```

Comment:
Not much point in keeping `AbstractExecutor` now. All other executors were superseded by SequenceExecutor.